### PR TITLE
WIP: Add section on how backends are queried

### PR DESCRIPTION
**Pull request by DennisPeriquet**
Ideas on adding images taken from [Egli's PR](https://github.com/openshift/ci-docs/pull/258).

TODO: fix the link that gets a 404; see my comment.

### CODE REVIEW

File: content/en/docs/release-oversight/disruption-testing.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ To check for disruptions while upgrading OCP clusters
 * Which creates a new [Chaosmonkey](https://github.com/neisw/origin/blob/59599fad87743abf4c84f05952552e6d42728781/vendor/k8s.io/kubernetes/test/e2e/chaosmonkey/chaosmonkey.go#L48) and [executes](https://github.com/neisw/origin/blob/59599fad87743abf4c84f05952552e6d42728781/vendor/k8s.io/kubernetes/test/e2e/chaosmonkey/chaosmonkey.go#L78) the disruption monitoring tests and the disruption
 * The [backendDisruptionTest](https://github.com/neisw/origin/blob/0c50d9d8bedbd2aa0af5c8a583418601891ee9d4/test/extended/util/disruption/backend_sampler_tester.go#L34) is responsible for
   * Creating the event broadcaster, recorder and monitor
-  * Attempting to query the backend and timing out after the max interval (1 second typically)
+  * [Attempting to query the backend](backend_queries.md) and timing out after the max interval (1 second typically)
```
[the remark this answers is not in the document]
Comment (neisw) [reply content/en/docs/release-oversight/disruption-testing.md:41]: try
[Attempting to query the backend](../backend_queries)

[the remark this answers is not in the document]
Comment (dperique) [reply content/en/docs/release-oversight/disruption-testing.md:41]: Thank,I'll give it a whirl soon.

**Comment by eggfoobar** [comment]
This is great, in the PR I just opened up, I thought it might make more sense to split the disruption info into its own sub category. Do you guys think we should merge that in, and move this file under that folder? https://github.com/neisw/ci-docs/pull/2

**Comment by neisw** [comment]
> This is great, in the PR I just opened up, I thought it might make more sense to split the disruption info into its own sub category. Do you guys think we should merge that in, and move this file under that folder? #2

Yes, I just merged in your changes Egli.  Dennis can update his branch and we can get that refactored and merged in as well.

**Comment by DennisPeriquet** [comment]
my branch has been updated with the latest on [trt-237-disruption](https://github.com/neisw/ci-docs/tree/trt-237-disruption)

**Pull request merged**